CIPs/cip-xxxx.md

```diff
@@ -0,0 +1,128 @@
+
+An on-chain exchange proposal does not need to be created when the forum post is created if the exchanger is hoping to gauge opinions before locking funds in GrandaMento. However, the exchange proposal should be submitted on-chain prior to achieving rough consensus to allow community members to verify the on-chain exchange proposal for themselves. 
+
+#### Forum post title format
+
+```
+GrandaMento Exchange Proposal <on-chain proposal ID> Discussion: <sell amount> <sell token> for <buy amount> <buy token>
+```
+For example: `GrandaMento Exchange Proposal 1 Discussion: 500,000 CELO for 1,500,000 cUSD`.
+
+If the proposal has not been created on-chain yet, the proposal ID and buy amount can be omitted for the time being, though it should be added later upon exchange proposal creation. For example: `GrandaMento Exchange Proposal <TBD> Discussion: 500,000 CELO for cUSD`.
+
+#### Forum post contents
+```
+1. What token and quantity are you selling, and what token are you buying?
+
+2. Who are you?
+
+3. Why do you want to execute the exchange?
+// Include the intended use of bought funds.
+
+4. What is the on-chain proposal ID and the on-chain buy amount?
+// Detailed info can be found by running this command and sharing the output:
+//   `celocli grandamento:show --proposalId <proposal-id>`
+// If you are planning to create the on-chain exchange proposal after
+// fielding opinions from the community, please mention so here.
+
+--- Only if selling CELO ---
+5. How did you acquire the CELO?
+// For example, via the Celo Foundation, Mento, a centralized exchange, OTC, etc.
+
+6. At what price did you acquire the CELO?
+```
+
+For extra visibility, after creating the forum post, the exchanger should share the link to the forum post in the `#celo-governance` Discord channel.
+
+### Achieving rough consensus
+
+[Rough consensus](https://en.wikipedia.org/wiki/Rough_consensus) in favor of an exchange proposal is expected prior to the approver multisig approving an exchange proposal. 51% of discussion participants in agreement is seen as worse than rough consensus, while 99% is seen as better than rough consensus. Rough consensus is seen as sufficient for approval because community members are able to create a governance proposal to veto an approved exchange proposal. "Achieving" rough consensus is defined as the approver multisig deciding to approve the exchange proposal as a result of rough consensus being in favor of the exchange proposal.
+
+A forum post must exist for at least one week prior to rough consensus being achieved. The on-chain proposal must be submitted and the corresponding on-chain proposal ID must be shared in the forum post for at least 4 days prior to achieving rough consensus. This gives community members the opportunity to independently verify the on-chain exchange proposal.
```

Comment:
So the proposer has to hold CELO for one week? Wouldn't the fluctuations in price during that period potentially outweigh the benefit of lower slippage with Granda Mento? I guess we'll see if anyone uses Granda Mento.

Reply:
@Pinotio let's reserve content discussions for the github discussions link as soon as @tkporter creates one